business/istio_config.go

```diff
@@ -662,6 +722,34 @@ func (in *IstioConfigService) GetIstioConfigDetails(namespace, objectType, objec
 		} else {
 			err = iErr
 		}
+	case EnvoyFilters:
+		if ef, iErr := in.k8s.GetEnvoyFilter(namespace, object); iErr == nil {
+			istioConfigDetail.EnvoyFilter = &models.EnvoyFilter{}
+			istioConfigDetail.EnvoyFilter.Parse(ef)
+		} else {
+			err = iErr
+		}
+	case AttributeManifests:
+		if am, iErr := in.k8s.GetAttributeManifest(namespace, object); iErr == nil {
+			istioConfigDetail.AttributeManifest = &models.AttributeManifest{}
+			istioConfigDetail.AttributeManifest.Parse(am)
+		} else {
+			err = iErr
+		}
+	case HttpApiSpecBindings:
+		if hb, iErr := in.k8s.GetHttpApiSpecBinding(namespace, object); iErr == nil {
+			istioConfigDetail.HttpApiSpecBinding = &models.HttpApiSpecBinding{}
+			istioConfigDetail.HttpApiSpecBinding.Parse(hb)
+		} else {
+			err = iErr
+		}
+	case HttpApiSpecs:
+		if hs, iErr := in.k8s.GetHttpApiSpec(namespace, object); iErr == nil {
+			istioConfigDetail.HttpApiSpec = &models.HttpApiSpec{}
+			istioConfigDetail.HttpApiSpec.Parse(hs)
+		} else {
+			err = iErr
+		}
```

Comment:
I believe we do want to stll users to modify this new objects using, right?
If so, I guess, there is need to update the method `modifyIstioConfigDetail` (around L#92)